lib/applicationinsights.json

```diff
@@ -0,0 +1,34 @@
+{
+  "connectionString": "${APP_INSIGHTS_CONNECTION_STRING}",
+  "role": {
+    "name": "idam-health-checker"
+  },
+  "preview": {
+    "roleNameOverrides": [
+      {
+        "httpPathPrefix": "/health",
+        "roleName": "idam-health-checker (health)"
+      }
+    ],
+    "sampling": {
+      "overrides": [
+        {
+          "telemetryType": "request",
+          "attributes": [
+            {
+              "key": "http.url",
+              "value": "https?://[^/]+/health.*",
```

Comment:
this probably needs to be /admin/health too

Reply:
Right changed it